<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">opacity:0.5;
background-color: rgb(255, 255, 255,100);
border-image: url(:/images/resource/images/2.png);
border-radius:15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="setWindowOpacity" stdset="0">
    <string>0.7</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(216, 216, 216,180);
border-radius:25px;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px</string>
    </property>
    <property name="placeholderText">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>密码：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton{
	
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border:1px solid rgb(0,0,0);
	border-radius:8px;
}
#pushButton:hover{
	
	background-color: rgb(158, 158, 158);
	
	color: rgb(255, 255, 255);
}
#pushButton:pressed{
	padding-top:5px;
	padding-left:5px;
}</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_2{
	
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
	border:1px solid rgb(0,0,0);
	border-radius:8px;
}
#pushButton_2:hover{
	
	background-color: rgb(158, 158, 158);
	
	color: rgb(255, 255, 255);
}
#pushButton_2:pressed{
	padding-top:5px;
	padding-left:5px;
}</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
